apps/SageAccountWeb/src/components/OAuthClientManagementPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { SourceAppLogo } from './SourceApp'
+import { StyledOuterContainer } from './StyledComponents'
+import { Box, Paper, Typography } from '@mui/material'
+import { BackButton } from './BackButton'
+import { OAuthManagement } from 'synapse-react-client/dist/containers/oauth/OAuthManagement'
+
+export const OAuthClientManagementPage = () => {
+  return (
+    <StyledOuterContainer className="OAuthClientManagementPage">
+      <Paper
+        className="SageResourcesPage"
+        sx={{
+          margin: '0 auto',
+          width: '900px',
+          '& .SourceAppLogo': {
+            display: 'block',
+          },
+        }}
+      >
+        <BackButton to={'/authenticated/myaccount'} />
+        <Box sx={{ padding: '60px' }}>
+          <SourceAppLogo sx={{ display: 'block' }} />
```

Comment:
I think it's kind of unusual to show the source app logo on this page, since (if it's not Synapse) the client dev won't be integrating with that app, and the docs/audience for OAuth clients will likely always be Synapse-specific.